### ISSUE TITLE
Code Table Request - U. S. National Park Service catalog

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#5969]
## Instructions

This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions.

Reviewing documentation before proceeding will result in a more enjoyable experience.

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step.

All [U. S. National Park Service catalog](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type#u__s__national_park_service_catalog) should be replaced with other ID type = ~~other identifier~~ institutional catalog number and issued by agent ~~[United States National Park Service](https://arctos.database.museum/agent/40)~~ [United States National Park Service Catalog](https://arctos.database.museum/agent/21347816)

**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos. 

**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable.

**Context**: Describe why this new value is necessary and existing values are not.

**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure.

**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm

**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking.

**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions. 

**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

## Approval

All of the following must be checked before this may proceed.

The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality).

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc. 

1. Can a suitable solution be found here? If not, proceed to (2)
2. Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)
3. Take the discussion to a monthly Arctos Working Group meeting for final resolution.

## Implementation

Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made.

Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel. 

Make changes as described above. Ensure the URL of this Issue is included in the definition. 

Close this Issue.

**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result.

## Special Exemptions

In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases.

1. Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step.
2. The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology.

**Comment by dustymc** [comment]
~~I will plan on proceeding with this about 2023-03-28 if there are no objections.~~

I will proceed immediately upon approval of each of the involved collections.

Data:
[temp_u_s_national_park_service_catalog.csv.zip](https://github.com/ArctosDB/arctos/files/10971924/temp_u_s_national_park_service_catalog.csv.zip)


Summary:
|guid_prefix|numrecs|
|-----------|--------|
|DMNS:Mamm|30|
|KWP:Ento|1|
|MSB:Bird|29|
|MSB:Mamm|11370|
|MSB:Para|1|
|MVZ:Mamm|1|
|UAM:Alg|1|
|UAM:Arc|3578|
|UAMb:Herb|14842|
|UAM:Ento|56111|
|UAM:ES|1186|
|UAM:Herb|19192|
|UAM:Inv|1|
|UAM:Mamm|5786|
|UAMObs:Ento|5|
|UCM:Bird|25|
|UCM:Fish|4|
|UCM:Herp|52|
|UCM:Mamm|193|
|UMNH:Mamm|1|
|UTEP:ES|1|
|UWBM:Mamm|2682|


Users:
@ebraker
@mkoo
@campmlc
@amgunderson
@DerekSikes
@mvzhuang
@cjconroy
@jebrad
@AdrienneRaniszewski
@acdoll
@jldunnum
@jrdemboski
@msbparasites
@mlbowser
@kderieg322079
@StefanieBond
@catherpes
@sjshirar
@jandreslopez


See also https://github.com/ArctosDB/arctos/issues/5771

**Comment by jebrad** [comment]
UWBM:Mamm approves.  Thank you.

**Comment by campmlc** [comment]
Suggest using "catalog number" instead of other identifier as ID name for these, to distinguish for Park Service accession number in separate issue. I am not convinced that NPS catalog and NPS accession work as an agent names - these are clearly IDs, and will be considered as such by the wider community. This should be discussed further.

**Comment by dustymc** [comment]
>This should be discussed further.

That's why it's marked as it is.

Yes NPS accn and catnum are different THINGS, I'm not sure how to best deal with that but it does need somehow acknowledged/maintained.

I think all of these (not that that will be possible) should be attributed to individual parks as well, but I'm not 100% sure how that works either. I'm hoping someone from NPS will show up and help us understand the functionality of these things.

**Comment by campmlc** [comment]
Yes, it would be ideal to have the issued by agent be the actual park, since they each issue their own IDs. And ID type as catalog number. See my earlier example: https://arctos.database.museum/guid/MSB:Mamm:222222

**Comment by jebrad** [comment]
I assumed these numbers would be perfect as "Institutional catalog numbers", but I trusted that Dusty had a good reason to want to make them Other Identifier instead.
Having the prefix "NPS Number:" always attached to the number seems important, to avoid confusion about where this number is stored, and as long as that prefix stays then this change will work for me, and that's why I approved (also I read this as he needed approval rather than discussion)

Only some of my records from National Parks include NPS numbers, not all of them, and I only ever use the NPS Catnum: never the Accession number (out of simplicity + not knowing function of NPS accession numbers).

The only time I ever use NPS numbers is when a given Park sends me a request for their annual inventory.  Many just send me a list of (eg) MORA numbers for me to go find, some NP curators seem reluctant to use or acknowledge any other ID number.

BUT -  I also thought NPs all had standard acronyms, eg MORA or OLYM numbers (Mount Rainier NP, Olympic NP).    But now I see that MSB:222222 has YNP instead of YELL for Yellowstone? aye aye aye.

**Comment by campmlc** [comment]
Hold pending resolution of other ID type. Given this is one of at least two identifiers issued by NPS, changing to other identifier is inappropriate. Propose using "catalog number" or institutional catalog number as ID type.

**Comment by kderieg322079** [comment]
I agree with @campmlc. [Other identifier is defined](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type#other_identifier), in part, as "various values that are not known to be of a more defined identifier type" and the NPS catalog and NPS accession are two distinct and specific identifier types. I think using "institutional catalog number" as ID type, as Mariel recommended, would be reasonable except it still doesn't address what we do with NPS accession numbers. I have yet to include them with NPS specimens at UMNH, but would like to just to have more complete data on our NPS specimens and be able to find them more easily when the parks do their inventories. Sometimes they only provide NPS catalog & accession without our catalog numbers, so I have to track them down based on the NPS identifiers.

**Comment by Jegelewicz** [comment]
> I also thought NPs all had standard acronyms, eg MORA or OLYM numbers (Mount Rainier NP, Olympic NP). But now I see that MSB:222222 has YNP instead of YELL for Yellowstone? aye aye aye.

I think this is true and you can find many of them on GRSciColl

I added those when I found them to the Park Agents - see https://arctos.database.museum/agent/1014744 and note the aka and the urls

**Comment by cjconroy** [comment]
Yosemite is YOSE

**Comment by Jegelewicz** [comment]
> Yosemite is YOSE

added aka

**Comment by Jegelewicz** [comment]
Working on data for UWBM:Mamm and in this case, I think it might make sense to keep the type. One can use the type to find ALL National Park catalog numbers and the issuer to find numbers associated with certain parks. From https://arctos.database.museum/guid/UWBM:Mamm:77941 for example:

Relationship | IssuedBy | Identifier | Type 
-- | -- | -- | -- 
self | [John Day Fossil Beds National Monument](https://arctos.database.museum/agent/21348187) | JODA 10555 | U. S. National Park Service catalog

This is how I plan to recommend @jebrad update all of the UWBM:Mamm NPS numbers. Please advise if it does not make sense.

**Comment by Jegelewicz** [comment]
FWIW - I realize that the number prefix designates the park, but that is more easily mangled (or left off) than an issuing agent.

**Comment by campmlc** [comment]
Absolutely support this. It is how I wish we would handle all similar ID types, such as zoo local IDs. The issuer would be the specific zoo etc.

**Comment by amgunderson** [comment]
I am revisiting this issue to again oppose the use of the "issued by" field. I do not support this. Do not add any "issued by" agent to any UAM:Mamm specimen identifier. And do not change park service numbers other IDs.
